src/index.js

```diff
@@ -11,13 +11,18 @@ module.exports = function emberCliUpdate(options) {
   let from = options.from;
   let to = options.to;
   let ignoreConflicts = options.ignoreConflicts;
+  let packageVersion = getPackageVersion('.');
+
+  if (!packageVersion) {
+    console.error('Ember CLI was not found in this project\'s package.json');
```

Comment:
All console logging happens in bin/ember-cli-update.js. We should return a rejected promise here.

Reply:
Updated